examples/verify_addresses_from_csv/verify.py

```diff
@@ -0,0 +1,70 @@
+# Usage
+# python verify.py input.csv
+
+import csv
+import lob
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath(__file__+'../../../..'))
+
+lob.api_key = 'test_fc26575412e92e22a926bc96c857f375f8b'
+
+skipFirstLine = True
+
+# Column indices in CSV
+address1 = 0
+address2 = 1
+city = 2
+state = 3
+postcode = 4
+country = 'US'
+
+try:
+    sys.argv[1]
+except IndexError:
+    print("Please provide an input CSV file as an argument.")
+    print("Usage: python verify.py <csv-input>")
+    sys.exit(1)
+
+# Open input files
+inputFile = open(sys.argv[1], 'rU')
+csvInput = csv.reader(inputFile)
```

Comment:
Can we use `csv.DictReader` here just like we do in the letters example? This will also remove the `skipFirstLine` option I believe since it'll be done by the `csv.DictReader`.